styles.css

```diff
@@ -0,0 +1,542 @@
+.main__button {
+  width: 44%;
+  font-size: 13px;
+  line-height: 36px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #e51937;
+  background: transparent;
+  border: 2px solid #e51937;
+  text-align: center;
+  transition-duration: 0.4s;
+}
+
+.main__button:hover {
+  background: #e51937;
+  color: #000;
+}
+
+/* блок со стрелкой вниз */
+.main__vector {
+  text-align: center;
+}
+
+/* Стрелка вниз */
+.vector {
+  border: solid black;
+  border-width: 0 3px 3px 0;
+  border-color: #e51937;
+  display: inline-block;
+  padding: 10px;
+}
+
+.down {
+  transform: rotate(45deg);
+  -webkit-transform: rotate(45deg);
+}
+
+@media (max-width: 1024px) {
+  .body {
+    margin: 0;
+    max-width: 1024px;
+  }
+
+  .header__nav-link {
+    display: none;
+  }
+
+  .header__menu {
+    width: 30px;
+  }
+
+  .main__wrap {
+    width: 90%;
+  }
+}
+
+@media (max-width: 768px) {
+  .body {
+    margin: 0;
+    max-width: 768px;
+  }
+
+  .header {
+    margin: 24px 33px 85px 6.32px;
+  }
+
+  .header__nav-link {
+    display: none;
+  }
+
+  .header__menu {
+    width: 36px;
+  }
+
+  .header__menu-burger {
+    width: 35.23px;
+    height: 22.02px;
+  }
+
+  .main__img-background {
+    width: 100%;
+    margin-top: 96px;
+  }
+
+  .main__img-tesla {
+    margin-left: 75px;
+    margin-top: 88px;
+    width: 94%;
+  }
+
+  .main__wrap {
+    width: 100%;
+  }
+
+  .main__inner {
+    margin: 0 auto;
+    width: 70%;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    margin-bottom: 41px;
+  }
+
+  .main__inner-road {
+    padding-left: 0;
+  }
+
+  .main__img-speed {
+    margin-top: 4px;
+    width: 65.92px;
+    height: 50.16px;
+    margin-right: 20px;
+  }
+
+  .main__img-road_dasktop {
+    margin: 28px 6px 0 0;
+    width: 59px;
+    height: 86px;
+    grid-column: 1;
+    grid-row: 1/3;
+  }
+
+  .main__text-big {
+    font-size: 50px;
+    line-height: 55px;
+  }
+
+  .main__text-norm {
+    font-size: 20px;
+    line-height: 49px;
+    padding-left: 9px;
+  }
+
+  .main__div-testdrive {
+    margin: 0 auto;
+    width: 70%;
+    flex-wrap: wrap;
+  }
+
+  .testdrive__wrapper {
+    width: 160px;
+  }
+
+  .main__img-testdrive {
+    fill: #e51937;
+    height: 30.37px;
+    width: 30.37px;
+  }
+
+  .main__link {
+    font-size: 16px;
+    line-height: 20px;
+    color: #e51937;
+  }
+
+  .main__button {
+    width: 220.84px;
+    line-height: 51px;
+    font-size: 16px;
+    background: #e51937;
+    color: #fff;
+  }
+
+  .main__vector {
+    margin-top: 52px;
+  }
+}
+
+@media (max-width: 560px) {
+  .body {
+    margin: 0;
+    max-width: 560px;
+  }
+
+  .main__h1 {
+    font-size: 75px;
+  }
+
+  .testdrive__wrapper {
+    margin-bottom: 30px;
+  }
+}
+
+@media (max-width: 375px) {
+  .body {
+    margin: 0;
+    max-width: 375px;
+  }
+
+  .header {
+    margin: 0 auto;
+    width: 100%;
+    align-items: center;
+  }
+
+  .header__logo-img {
+    width: 138px;
+    height: 42px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)